### DISCUSSION TITLE
TFT35

### DISCUSSION THREAD

**Discussion by StevieM63** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2661] [Q&A] [answered]
Please excuse my ignorance but am very much a novice with these things.  I have just fitted a BTT TFT35 touchscreen to my ender3 pro and all appears to work fine except every time I switch on my printer I get an error message that says 'Failed to enable bed levelling'.  I can clear this and all seems fine but how do I stop this message from occurring.  I have no level sensor fitted and have not as yet done any mesh bed levelling??
Also does any one know if there is a complete guide to all the menu options as I would like to see what is actually capable ?

**Comment by radek8** [comment, marked as answer]
Set in the cinfig.ini file.
auto_load_leveling:0

If you want to achieve good results, I recommend using some kind of leveling...

**Reply by StevieM63** [reply]
Thank you very much for that all now sorted.

**Comment by kisslorand** [comment]
...or you can switch this option off on the TFT itself going to Menu -> Settings -> Feature where you'll find the "Auto Save Load Leveling" option which can be switched on and off...

**Comment by StevieM63** [comment]
Thanks now I know 2 ways😀
